deploy/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
   db:
     # latest image as of writing
     image: postgres:12.17
-    env:
+    environment:
       POSTGRES_USER: "gpig"
       POSTGRES_PASSWORD: "heresapassword"
```

Comment:
I've had a look at https://docs.docker.com/compose/environment-variables/set-environment-variables/ and it says to use secrets for sending sensitive information like passwords, should we change `POSTGRES_PASSWORD` to a secret instead? 

We could change this to:
```
environment:
  POSTGRES_USER: "gpig"
  POSTGRES_PASSWORD_FILE: /run/secrets/POSTGRES_PASSWORD
secrets:
  - POSTGRES_PASSWORD
```

then at the bottom add the following:

```
secrets:
  POSTGRES_PASSWORD:
    file: POSTGRES_PASSWORD.txt
```

and would also need to create a file `POSTGRES_PASSWORD.txt` that contains our password

Reply:
We can switch to secrets mounting, yeah.

It will remove our one-click setup though, and I was planning to move to an environment file once we have a better idea of what our microservices look like.

Reply:
I agree with Laura, secrets are a bit overkill for local development.

Reply:
Let's kick this to our next agenda, this PR isn't urgent right now.